api/opds.py

```diff
@@ -0,0 +1,50 @@
+from nose.tools import set_trace
+from opds.opds_writer import OPDSFeed
+
+class DictionaryFeed(OPDSFeed):
+  SCHEMA_DEFINED_TERM = "http://schema.org/DefinedTerm"
+  TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ%z'
+
+  def __init__(self, word, url, language, definitions):
+    super().__init__(word, url)
+
+    self.language = language
+    self.feed["definitions"] = []
+
+    self.add_metadata()
+    self.add_definitions(definitions)
+
+  def add_metadata(self):
+    super().add_metadata()
+    self.metadata["title"] = "Definitions for %s" % self.title,
+    self.metadata["@type"] = self.SCHEMA_DEFINED_TERM
+    self.metadata["language"] = self.language
+    self.metadata["name"] = self.title
+
+  def get_feed(self):
+    return self.feed
+
+  def add_definitions(self, definitions):
+    '''Adds entries for each definition.
+    :param definitions: a list of definition objects with 'glosses', 'pos',
+      and 'tags' properties coming from Wiktionary.
+    '''
+    entries = [self.add_entry(definition) for definition in definitions]
+
+  def add_entry(self, definition):
+    '''Adds entries for each definition.
+    :param definition: an objects with 'glosses' and 'tags' properties
+    '''
+
+    definition_element = None
+    definitions = []
+    for d in definition.get("glosses", []):
+      definition_element = dict(
+        metadata=d,
+        tags=definition.get("tags", []),
+        pos=definition.get("pos", None)
+      )
```

Comment:
A couple things about `metadata`  here:

* `metadata` always needs to be a dictionary.
* `tags` and `pos` can go into that dictionary.
* I don't know where this is specified, but the Feedbooks OPDS feeds use a list of `subject`s inside `metadata` where you're using `tags`. I think it would be better to copy that. Just give your subjects the same `code` and `name`. Either make up a `scheme` or we can come up with one later.

![image](https://user-images.githubusercontent.com/662474/74951752-03c79000-53ce-11ea-8364-a725a30e6ab8.png)